src/app/helptext/sharing/afp/afp.ts

```diff
@@ -138,6 +138,12 @@ export const helptext_sharing_afp = {
   dialog_message: T("Enable this service?"),
   dialog_button: T("Enable Service"),
 
+  dialog_delete_title: T("Delete"),
+  dialog_delete_message: T(
+    "Stop sharing AFP (Apple File Protocol)? The sharing configuration\
+ will be deleted (but the underlying dataset will not)."
```

Comment:
I thought we came up with a wording for this already for the SMB service, but maybe not.
```suggestion
 will be deleted. Data in the former share dataset will not be affected."
```